.github/workflows/test_suite.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [ main, develop ]
   pull_request:
-    branches: [ main, develop ]
```

Comment:
@TomMelt to look into draft PRs. If not, add a comment here for how to ignore certain branches.